docs/openapi3.yaml

```diff
@@ -1591,6 +1591,11 @@ components:
           type: string
           description: Default webhook url to send live report statistics to, can be
             configured only by using API.
+        delay_runner_ms:
+          type: number
```

Comment:
add `minimum: 0` so it stays positive

can you please also add this to `minimum_wait_for_delayed_report_status_update_in_ms`.

Also I think we should add a minimum for runner_memory and runner_cpu

Reply:
done